### DISCUSSION TITLE
Error after updating to Pimcore X

### DISCUSSION THREAD

**Discussion by Hbitvof** [pimcore/pimcore#10873] [Q&A / Support]
Hi,

After successfully updating from 6.9 to x I can login into Pimcore, then I only see a white screens with running dots.
When looking at debugging mode I see the following errors:
![image](https://user-images.githubusercontent.com/41229269/142771355-f52eca4f-e8c1-4d54-adf8-e05112bf2af2.png)
and
![image](https://user-images.githubusercontent.com/41229269/142771330-edc9f033-4ee6-4b07-8ac6-b99280dbd4d6.png)

I can't find anything about this. Anyone know what's going wrong here?

**Comment by Hbitvof** [comment]
Anyone?

**Reply by dpfaffenbauer** [reply]
cache clear tried?
```
bin/console pimcore:cache:clear
```

**Reply by Hbitvof** [reply]
> 

yes, I did but that's not solving anything.

**Comment by dvesh3** [comment]
@Hbitvof run command `php bin/console assets:install public --symlink --relative` to re-install bundle assets and check file system permissions defined here https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/File_Permissions.html

**Reply by Hbitvof** [reply]
> @Hbitvof run command `php bin/console assets:install public --symlink --relative` to re-install bundle assets and check file system permissions defined here https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/File_Permissions.html

Tried both steps (After every upgrade I always set the permissions) but still the same result (also cleared cache after the first step).

**Comment by Hbitvof** [comment]
Anyone please?
